cli/plasmo/src/features/extension-devtools/common-path.ts

```diff
@@ -20,7 +20,10 @@ export const getCommonPath = (
     ? srcDirectory
     : projectDirectory
 
-  const buildDirectory = resolve(projectDirectory, "build")
+  const buildDirectory = resolve(
+    projectDirectory,
+    getFlag("--build-dir") || "build"
```

Comment:
`--build-dir` or `--build-path`? I'm fine with either, but we need to be consistent

Reply:
I changed the PR name before pushing the new change xD